src/web/components/Core/Dialog.scss

```diff
@@ -0,0 +1,68 @@
+.dialog-overlay {
+  background-color: rgba(255, 255, 255, 0.8);
+  position: fixed;
+  inset: 0;
+  animation: overlayShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.dialog-content {
+  background-color: white;
```

Comment:
Make sure you use theme variables rather than colours. --theme-background-content is probably fine here I guess.